docs/admin/user-management.rst

```diff
@@ -18,7 +18,9 @@ superusers that already exist in the CrateDB cluster. The `CREATE USER`_ and
 When CrateDB is started, the cluster contains one predefined superuser. This
 user is called ``crate``. It is not possible to create any other superusers
 
-Users cannot be backed up or restored.
+The definition of all users, including hashes of their passwords, is backed up
+together with the cluster's metadata when a snapshot is created, and it can be
+restored with the ``USERS`` option of the :ref:`sql-restore-snapshot` command.
```

Comment:
It will be also restored when using `RESTORE ALL`, see https://crate.io/docs/crate/reference/en/5.0/sql/statements/restore-snapshot.html. `RESTORE USERS` (or any other section) will restore the users **only**.
```suggestion
together with the cluster's metadata when a snapshot is created, and is 
restored when using the ``ALL`` or ``USERS`` clause of the 
:ref:`sql-restore-snapshot` command.
```